nucliadb/tests/ingest/fixtures.py

```diff
@@ -174,17 +173,12 @@ async def fake_node(indexing_utility, shard_manager):
 
 @pytest.fixture()
 def learning_config():
-    lconfig = LearningConfiguration(
-        semantic_model="multilingual",
-        semantic_threshold=None,
-        semantic_vector_size=len(V1),
-        semantic_vector_similarity="cosine",
-    )
-    with patch("nucliadb.ingest.service.writer.learning_proxy") as mocked:
-        mocked.set_configuration = AsyncMock(return_value=None)
-        mocked.get_configuration = AsyncMock(return_value=lconfig)
-        mocked.delete_configuration = AsyncMock(return_value=None)
-        yield mocked
+    from nucliadb_utils.settings import nuclia_settings
```

Comment:
Since this changes the behaviour of the fixture, maybe it needs a different name. THe most obvious would be `dummy_learning_services` or something like that.